lisa/operating_system.py

```diff
@@ -532,7 +532,7 @@ def _get_package_information(self, package_name: str) -> VersionInfo:
     def _get_version_info_from_named_regex_match(
         self, package_name: str, named_matches: Match[str]
     ) -> VersionInfo:
-        essential_matches = ["major", "minor", "build"]
+        essential_matches = ["major", "minor"]
```

Comment:
Why remove the build part? It may affect all other distros.

Reply:
It's not removing it, it's making it optional since sometimes it's missing, similar to patch. If you check the processing function below: build can be None. It was supposed to be optional, so this is more of a bug fix. I've run into package versions like '4.10' so making patch and build optional is needed.